comdirect_api/comdirect_client.py

```diff
@@ -1,28 +1,42 @@
 import requests
+import pickle
 
 from comdirect_api.auth.auth_service import AuthService
 from comdirect_api.service.account_service import AccountService
 from comdirect_api.service.depot_service import DepotService
 from comdirect_api.service.document_service import DocumentService
 from comdirect_api.service.report_service import ReportService
+from comdirect_api.service.order_service import OrderService
 
 
 class ComdirectClient:
 
-    def __init__(self, client_id, client_secret):
+    def __init__(self, client_id, client_secret, import_session=False):
         self.api_url = 'https://api.comdirect.de/api'
         self.oauth_url = 'https://api.comdirect.de'
-        self.session = requests.Session()
-        self.session.headers.update({
-            'Accept': 'application/json',
-            'Content-Type': 'application/json',
-        })
+        
+        if not import_session:
+            self.session = requests.Session()
+            self.session.headers.update({
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+            })
+            self.auth_service = AuthService(client_id, client_secret, self.session, self.api_url, self.oauth_url)
+        else:
+            with open(import_session, 'rb') as input:
+                self.session = pickle.load(input)
+                self.auth_service = pickle.load(input)
 
-        self.auth_service = AuthService(client_id, client_secret, self.session, self.api_url, self.oauth_url)
         self.account_service = AccountService(self.session, self.api_url)
         self.depot_service = DepotService(self.session, self.api_url)
         self.document_service = DocumentService(self.session, self.api_url)
         self.report_service = ReportService(self.session, self.api_url)
+        self.order_service = OrderService(self.session, self.api_url)
+
+    def session_export(self, filename = 'session.pkl'):
+        with open(filename, 'wb') as output:
+            pickle.dump(self.session, output, pickle.HIGHEST_PROTOCOL)
+            pickle.dump(self.auth_service, output, pickle.HIGHEST_PROTOCOL)
```

Comment:
I really like the idea of exporting the session 👍🏽  one minor suggestion:

I see that `import_session` can be false and a filepath-string if we want to import the session. What happens if the caller sets `import_session=True`? I think we need to catch that error, since the method signature suggests that boolean values are allowed.
Maybe in that case we could even use the default `session.pkl` filename to load the session from, since that is already the default in `session_export`.

Reply:
Hi alex-kn, thank you for your work on this repo! I like the idea to use the default filename for import_session=True and changed it that way.